iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/arithmetic_ops.mlir

```diff
@@ -3,7 +3,7 @@
 // CHECK-LABEL: @add_i32
 vm.module @my_module {
   vm.func @add_i32(%arg0: i32, %arg1: i32) {
-    // CHECK-NEXT: %0 = emitc.call "vm_add_i32"(%arg0, %arg1) : (i32, i32) -> i32
+    // CHECK-NEXT: %0 = emitc.call "vm_add_i32"(%arg0, %arg1) {args = [0 : index, 1 : index]} : (i32, i32) -> i32
```

Comment:
The conversion relied on a special case in emitc before. Operands get simply forwarded in the printer when encountering an empty args attribute.

In other words, these two lines are equivalent.